files/requirements.txt

```diff
@@ -13,3 +13,4 @@ psutil>=2.0.0
 ydiff>=1.2.0
 pexpect>=4.8.0
 ruamel.yaml>=0.16.10
+ansible>=2.7
```

Comment:
note that the file requirements.txt it is used to [install](https://github.com/vitabaks/postgresql_cluster/blob/master/roles/patroni/tasks/main.yml#L32) dependencies on the target database servers, I do not think that ansible needs to be installed there, it is not needed there.

Reply:
I apologize for my mistake. I misunderstood the purpose of the requirements.txt file and thought it was for the Ansible bastion. It seems that it's used to install dependencies on the target database servers, and you're right that Ansible doesn't need to be installed there.

I will make the necessary corrections tomorrow to resolve this issue.

Reply:
This situation highlights another reason to consider switching to collections😄. But on a more serious note, I understand the need to keep the configuration as simple and "KISS" (Keep It Simple, Stupid) as possible, and I will take this into account moving forward.

Reply:
Done and tested on gitpod with Makefile (working)

Reply:
the downside of this approach is the increase in the creation time of the gitpod working environment. 

Do we really need compliance with gitpod? Why can't we consider this as two independent options?